package.json

```diff
@@ -13,22 +18,19 @@
     "gatsby-plugin-sass": "^2.8.0",
     "gatsby-source-filesystem": "^2.8.1",
     "gatsby-transformer-remark": "^2.13.1",
+    "i18next": "^20.3.2",
+    "i18next-browser-languagedetector": "^6.1.2",
+    "i18next-http-backend": "^1.2.6",
+    "i18next-xhr-backend": "^3.2.2",
     "netlify-cms-app": "^2.14.8",
     "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
-    "uswds": "^2.10.0",
-    "@reach/skip-nav": "^0.15.2",
-    "@trussworks/react-uswds": "^2.0.0",
-    "classnames": "^2.3.1",
-    "i18next": "^20.3.2",
-    "i18next-browser-languagedetector": "^6.1.2",
-    "i18next-http-backend": "^1.2.6",
-    "i18next-xhr-backend": "^3.2.2",
     "react-i18next": "^11.11.0",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "uswds": "^2.10.0"
```

Comment:
Was downgrading `uswds` the fix?

Reply:
Not sure which package was the problem since I downgraded multiple at once - but it's possible that it was the uswds version in particular!